### PR TITLE
Adding 4.7 upgrade sanity tests for EUS

### PR DESCRIPTION
**Pull request by jhou1**
Adding tags to run upgrade-sanity tests for EUS upgrades. @liangxia @pruan-rht @JianLi-RH @dis016 PTAL PTAL

### CODE REVIEW

File: doc/prow/4.7/query_upgrade_sanity.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "case_query": "products.KEY:ocp AND NOT status:inactive AND type:testcase AND caseautomation.KEY:automated AND automation_script:cucushift AND NOT upstream.KEY:yes AND env_install_method.KEY:(__all__ ipi) AND caseimportance.KEY:critical AND version.KEY:4_7 AND NOT tags:(prod_only stage_only)"
```
Comment (liangxia) [inline doc/prow/4.7/query_upgrade_sanity.json:2]: Perhaps reflect ipi in the file name ?

Comment (jhou1) [reply doc/prow/4.7/query_upgrade_sanity.json:2]: Thanks, I'll correct the query and test cases, the query should include critical tests, regardless of installation methods.

Comment (jhou1) [reply doc/prow/4.7/query_upgrade_sanity.json]: Updated, the query result did not change, so no new tags added

**Pull request merged**